include/pokemon.h

```diff
@@ -398,31 +398,39 @@ struct SpeciesInfo /*0xC4*/
  /* 0x43 */ u8 backAnimId;
  /* 0x44 */ const union AnimCmd *const *frontAnimFrames;
  /* 0x48 */ const u32 *frontPic;
- /* 0x4C */ const u32 *frontPicFemale;
- /* 0x50 */ const u32 *backPic;
- /* 0x54 */ const u32 *backPicFemale;
- /* 0x58 */ const u32 *palette;
- /* 0x5C */ const u32 *paletteFemale;
- /* 0x60 */ const u32 *shinyPalette;
- /* 0x64 */ const u32 *shinyPaletteFemale;
- /* 0x68 */ const u8 *iconSprite;
+ /* 0x4C */ const u32 *backPic;
```

Comment:
Can we just remove the hex? Last time I checked they were all wrong.

Reply:
Done.